examples/research_projects/onnxruntime/text_to_image/train_text_to_image.py

```diff
@@ -412,6 +481,13 @@ def main():
 
     if args.gradient_checkpointing:
         unet.enable_gradient_checkpointing()
+        vae.enable_gradient_checkpointing()
+    else:
```

Comment:
Note that gradient checkpointing doesn't necessarily mean that the model is training

Reply:
How exactly is fine-tuning the VAE done in `train_text_to_image`? It seems like the issue  #1883 wants to enable gradient checkpointing specifically for training VAE independently so should we add in some sort of argument?